### DISCUSSION TITLE
[Question][Hw3] LLVM opt-10 is not installed in the prebuilt docker image.

### DISCUSSION THREAD

**Discussion by askme143** [prosyslab-classroom/cs424-program-reasoning#318] [Q&A]
I'm using a provided docker image to do homework.
According to the Makefile in the 'test' directory, `opt-10` is used to apply mem2reg optimization. However,`opt-10` is not installed in the pre-built docker image.
Is it okay to edit the Makefile to use `opt-13` instead? HW1 also used `opt-13` instead of `opt-10`.

```
make: opt-10: Command not found
make: *** [Makefile:12: example5.ll] Error 127
```

**Comment by Hhro** [comment]
Oh, that was my mistake.
It should be `opt-13` not `opt-10`.
Thanks.

**Comment by askme143** [comment]
Thanks.
